ui/apps/platform/src/Containers/Collections/CollectionsFormPage.tsx

```diff
@@ -236,7 +198,16 @@ function CollectionsFormPage({
                 isInlineDrawer={isLargeScreen}
                 isDrawerOpen={isDrawerOpen}
                 toggleDrawer={toggleDrawer}
-                onSubmit={onSubmit}
+                onSubmit={(collection) =>
+                    onSubmit(collection)
+                        .then(() => {
+                            history.push({ pathname: `${collectionsBasePath}` });
```

Comment:
Specific functionality for the `CollectionsFormPage` was pulled out and handled when the new promise resolves/rejects since the behavior will be different in the vuln management modal.